File: client/src/main/java/in/clayfish/printful/models/includable/Country.java
package in.clayfish.printful.models.includable;

import java.io.Serializable;
import java.util.List;

/**
 * @author shuklaalok7
 * @since 24/12/2016
 */
public class Country implements Serializable {

    /**
     * Country code
     */
    private String code;

    /**
     * Country name
     */
    private String name;

    /**
     * List of states
     */
    private List<State> states;

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public List<State> getStates() {
        return states;
    }

    public void setStates(List<State> states) {
        this.states = states;
    }
}

File: client/src/main/java/in/clayfish/printful/models/includable/Country.kt
package `in`.clayfish.printful.models.includable

import java.io.Serializable

/**
 * @author shuklaalok7
 * @since 24/12/2016
 */
data class Country(
        /**
         * Country code
         */
        var code: String? = null,

        /**
         * Country name
         */
        var name: String? = null,

        /**
         * List of states
         */
        var states: List<State>? = null
) : Serializable {

    constructor() : this(null, null, null)
    constructor(code: String?, name: String?) : this(code, name, listOf())

}
